### PR TITLE
Feature/zs add moodle docker stack

### LINKED ISSUE #18: Moodle Docker Compose

### PR DESCRIPTION
**Pull request by zanssa**
Resolves #18

### CODE REVIEW

**Comment by thatnerdjosh** [comment]
This is great, I actually like your idea of basing on master, I'll merge the develop branch to master so the work is easier to find... no need to have a develop and master at this point

File: moodle/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  moodle-data:
+    driver: local
+networks:
+    moodle-network:
+services:
+  mariadb:
+    image: 'mariadb:latest'
+    container_name: mariadb
+    environment:
+      MYSQL_ROOT_PASSWORD: YOUR_PASSWORD
```
Comment (thatnerdjosh) [inline moodle/docker-compose.yml]: Could we follow the same pattern for env vars that I have in the elgg docker compose? (`env_file`)

Thanks!

File: moodle/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    image: 'bitnami/moodle:latest'
+    container_name: moodle
+    environment:
+      MOODLE_DATABASE_HOST: mariadb
+      MOODLE_DATABASE_USER: moodle
+      MOODLE_DATABASE_NAME: moodle
+      MOODLE_DATABASE_PASSWORD: YOUR_PASSWORD
+    ports:
+      - '80:8080'
+      - '443:8443'
```
Comment (thatnerdjosh) [inline moodle/docker-compose.yml]: How is TLS configured? Should we have a central NGINX proxy configured to terminate TLS and reverse proxy our apps?

**Review by thatnerdjosh — CHANGES_REQUESTED** [review]

File: moodle/docker-compose.yml (hunk 2 above)
Comment (zanssa) [reply moodle/docker-compose.yml]: @thatnerdjosh , That's a good question! Actually, I wasn't told to take care of this and I haven't seen this being done in other apps.

File: moodle/docker-compose.yml (hunk 1 above)
Comment (zanssa) [reply moodle/docker-compose.yml]: @thatnerdjosh , Sure! will try that out.

**Pull request merged**